lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -46,6 +51,7 @@ $nx-global-sidebar-separator-color: $nx-grey-900;
   display: flex;
   height: $nx-global-sidebar-logo-height;
   margin-bottom: $nx-spacing-xl;
+  padding: 0 $nx-spacing-l;
```

Comment:
Could/should be margin?

Reply:
> Could/should be margin?

It could be but we use padding for l/r spacing for every other container in the sidebar so I was keeping it consistent. I can't see a reason it matters.

Reply:
> It could be but we use padding for l/r spacing for every other container in the sidebar so I was keeping it consistent. I can't see a reason it matters.

I guess it comes down to whether we'd rather be consistent with the other children within this component, or consistent with the way we do things more broadly, only deviating where really necessary.  I don't feel strongly about it.

Reply:
> > It could be but we use padding for l/r spacing for every other container in the sidebar so I was keeping it consistent. I can't see a reason it matters.
> 
> I guess it comes down to whether we'd rather be consistent with the other children within this component, or consistent with the way we do things more broadly, only deviating where really necessary. I don't feel strongly about it.

I am content with consistency within the component, but likely nobody but you and I will notice that consistency (or lack thereof).